source/brailleTables.py

```diff
@@ -82,325 +97,470 @@ def listTables():
+	# braille settings dialog.
+	addTable("ru.ctb", _("Russian braille for computer code"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("ru-ru-g1.utb", _("Russian grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("sa-in-g1.utb", _("Sanskrit grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("Se-Se.ctb", _("Swedish 8 dot computer braille"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("Se-Se-g1.utb", _("Swedish grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("sk-g1.ctb", _("Slovak grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("sl-si-comp8.ctb", _("Slovenian 8 dot computer braille"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("sl-si-g1.utb", _("Slovenian grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("sr-g1.ctb", _("Serbian grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("ta-ta-g1.ctb", _("Tamil grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("te-in-g1.utb", _("Telugu grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("tr.ctb", _("Turkish grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("uk.utb", _("Ukrainian"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("unicode-braille.utb", _("Unicode braille"), output=False)
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("vi-g1.ctb", _("Vietnamese grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("zhcn-g1.ctb", _("Chinese (China, Mandarin) grade 1"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("zhcn-g2.ctb", _("Chinese (China, Mandarin) grade 2"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("zh-hk.ctb", _("Chinese (Hong Kong, Cantonese)"))
+	# Translators: The name of a braille table displayed in the
+	# braille settings dialog.
+	addTable("zh-tw.ctb", _("Chinese (Taiwan, Mandarin)"))
+
+	# Add custom tables.
+	import os
+	from logHandler import log
+	customDirs = _getCustomTablesDirs()
+	tablesDirs[:0] = customDirs
+	# Reversing the order ensures that user defined configuration takes
+	# precedence over addons.
+	log.warning(f"customDirs={customDirs!r}")

```

Comment:
I guess this can be removed?

Reply:
Oups, damn dev log... I'm removing this one right away!
It is probably worth adding a debug log of the whole `tablesDirs` list passed at `louisHelper` initialization though.

Reply:
Dev log removed as of ea43238.
Debug log added as of af30589.